mmdet3d/models/backbones/pointnet2_sa_msg.py

```diff
@@ -63,7 +63,9 @@ def __init__(self,
         self.out_indices = out_indices
         assert max(out_indices) < self.num_sa
         assert len(num_points) == len(radii) == len(num_samples) == len(
-            sa_channels) == len(aggregation_channels)
+            sa_channels)
+        if aggregation_channels is not None:
```

Comment:
I just wonder if this change is necessary? You only judge if `aggregation_channels` is `None` here. However in [this line](https://github.com/open-mmlab/mmdetection3d/blob/cab70db5c67178aa90c7bf3aaf1ef844fe39da32/mmdet3d/models/backbones/pointnet2_sa_msg.py#L108), we still index from `aggregation_channels` and it will cause an error if `aggregation_channels is None` right?

In my opinion, I think the `aggregation_channels` should not be `None`. Instead it should be a list of `None`? So we should keep the original assert condition.